lib/db/redis.js

```diff
@@ -73,6 +75,29 @@ RedisDB.prototype.tail = function () {
   return P.resolve('0')
 }
 
+RedisDB.prototype.set = function (key, value, ttl) {
```

Comment:
Naming these just get/set/del seems a little too terse IMHO, especially since they appeat to have subscription-specific logic in them.  `getSub` et al?